contrib/cirrus/lib.sh

```diff
@@ -63,7 +63,7 @@ FEDORA_BASE_IMAGE="fedora-cloud-base-30-1-2-1559164849"
 PRIOR_FEDORA_BASE_IMAGE="fedora-cloud-base-29-1-2-1559164849"
 BUILT_IMAGE_SUFFIX="${BUILT_IMAGE_SUFFIX:--$CIRRUS_REPO_NAME-${CIRRUS_BUILD_ID}}"
 # IN_PODMAN container image
-IN_PODMAN_IMAGE="quay.io/libpod/in_podman:latest"
+IN_PODMAN_IMAGE="quay.io/haircommander/in_podman:latest"
```

Comment:
should this be reverted?

Reply:
@cevich as I understand it, in_podman was failing because it didn't have the correct conmon, and wouldn't until this PR was merged into master. Since conmon is being built from package now, once that package goes downstream, will in_podman pick it up? or is in_podman still building conmon from source?